README.md

```diff
@@ -5,17 +5,59 @@
 [![Dependency Status](https://david-dm.org/silvermine/eslint-plugin-silvermine.png)](https://david-dm.org/silvermine/eslint-plugin-silvermine)
 [![Dev Dependency Status](https://david-dm.org/silvermine/eslint-plugin-silvermine/dev-status.png)](https://david-dm.org/silvermine/eslint-plugin-silvermine#info=devDependencies&view=table)
 
-
 ## What?
 
-Shareable [eslint](http://eslint.org/) plugins that are used by our shareable config rules. See [eslint-config-silvermine](https://github.com/silvermine/eslint-config-silvermine/) for more details.
-
+Shareable eslint plugin for use by eslint-config-silvermine
 
 ## Why?
 
 Because we need it. Whitespace errors are evil. As are the other hundreds of types of errors this protects us from.
 
+## Installation
+
+You'll first need to install [ESLint](http://eslint.org):
+
+```
+$ npm i eslint --save-dev
```

Comment:
Why does this line use the shortened `i` form when the next line down is inconsistent and use the longer `install` form? Please use the full `install` form in both places.